src/utils.js

```diff
@@ -64,3 +64,25 @@ export function isEmpty(value) {
   }
   return false;
 }
+
+// We can have initial values undefined, but input makes them empty strings after edit/clear. We will consider those equal.
+
+const withException = (original, current) =>
+  (original === undefined || original === null) && current === ''
+    ? true
+    : undefined;
+
+function baseIsEqual(value, other) {
+  if (value === other) {
+    return true;
+  }
+
+  return false;
+}
+
+function isEqualWith(value, other, customizer) {
+  const result = customizer ? customizer(value, other) : undefined;
+  return result === undefined ? baseIsEqual(value, other) : !!result;
```

Comment:
```js
return result === undefined ? value === other : !!result
```

No need to have `baseIsEqual`

Reply:
agree, now there is no need for that (i had something extra in there at some point)